test/helper.py

```diff
@@ -19,7 +19,7 @@ def __init__(self):
 
     def login_as(self, username):
         password = self._make_password(username)
-        api = UserAPI(self.config.api_server, username, password).login()
+        api = UserAPI(self.config.api_server, username, None, password).login()
```

Comment:
If you wanted to avoid the extra param you could use the existence of a '@' to know whether an email address or username was being used.

Reply:
The UserAPI will need the email most of the time so it can register. This time it doesn't need it as it is just logging in.